firebase-firestore/src/main/java/com/google/firebase/firestore/FirebaseFirestore.java

```diff
@@ -448,9 +448,27 @@ private <ResultT> Task<ResultT> runTransaction(
   @NonNull
   public <TResult> Task<TResult> runTransaction(
       @NonNull Transaction.Function<TResult> updateFunction) {
+    return runTransaction(TransactionOptions.DEFAULT, updateFunction);
+  }
+
+  /**
+   * Executes the given updateFunction and then attempts to commit the changes applied within the
```

Comment:
Wrap updateFunction in {@code} block. Same to transactionOptions.

Reply:
I'll update the other functions as well to stay consistent.